README.md

```diff
@@ -46,11 +47,13 @@ go run main.go  # or go build main.go
 
 Features are identified by a _collection name_ and _feature primary key_ pair.
 
-- Collections: http://localhost:9000/api/collectionNames
-- Features primary keys: http://localhost:9000/api/featurePks
-- Features from a single collection: http://localhost:9000/api/collection
-- Single feature from a single collection: http://localhost:9000/api/feature
-- Create new, filtered, temporary collection: http://localhost:9000/api/feature_set
+- API landing: http://localhost:9000/
+- API: http://localhost:9000/api
+- API landing: http://localhost:9000/conformance
```

Comment:
This is not the same as the landing page, it's links to indicate what features the server implements.
Something like:
`Feature sets implemented: http://localhost:9000/conformance`
or
`Conformance: http://localhost:9000/conformance`